bundle.Dockerfile

```diff
@@ -5,7 +5,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=node-healthcheck-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```

Comment:
Do we need to include `alpha` as well so that existing deployments have a path to the new channel?
Or is that not how it works?